critiquebrainz/db/review.py

```diff
@@ -322,7 +334,7 @@ def list_reviews(*, inc_drafts=False, inc_hidden=False, entity_id=None,
                  offset=None):
     """Get a list of reviews.
 
-    This function provides several filters that can be used to select a subset of reviews.
+    This function provides several  filters that can be used to select a subset of reviews.
```

Comment:
Hi, it seems a space was added here by mistake

Reply:
oh, yeah, you're right, I'll remove that right away

Reply:
sorry about making more problems with one of the commits; I'm on a school computer and not having fun using the in-browser editor